function/services/application-mapper/application-question.test.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
```

Comment:
Tests here are fine, but you're still falling below the coverage threshold on this file.